### PR TITLE
Time and weather

### PR DESCRIPTION
**Pull request by Sharmaryan**
Screenshot (99)
Screenshot (101)
Screenshot (102)

- random image on page refresh
- user name saves to local storage
- time with user greet

### CODE REVIEW

**Review by chanchal-panpaliya — APPROVED** [review]
The UI looks good. I think you can merge it.

File: public/index.html [hunk 1]
```diff
@@ -9,9 +9,11 @@
       name="description"
       content="Web site created using create-react-app"
     />
-  
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Fruktur&family=Roboto:ital,wght@0,100;1,100;1,900&display=swap" rel="stylesheet">
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-   
+   <!-- <script src="https://cdn.tailwindcss.com"></script> -->
```
Comment (ravikumar0403) [inline public/index.html]: remove commented code if not required

File: src/components/UserOnboarding.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        }
+      }
+
+      if (hour < 12) {
+        dispatch({ type: "SET_GREET", payload: "Good Morning" });
+      }
+
+      let minute = today.getMinutes();
+      if (minute < 10) minute = "0" + minute;
+      dispatch({ type: "SET_TIME", payload: { hour, minute } });
```
Comment (ravikumar0403) [inline src/components/UserOnboarding.js:33]: You can use `toLocaleTimeString` method to get the formatted time
like this `const [hour, minute, second] = today.toLocaleTimeString().split(":")`

**Review by ravikumar0403 — APPROVED** [review]
I've left a couple of comments, Everything else looks good

File: src/components/UserOnboarding.js (hunk 2 above)
Comment (Sharmaryan) [reply src/components/UserOnboarding.js:33]: sure, I will be refactoring time part a little , at that time I will do it.

**Pull request merged**